manage_breast_screening/core/jinja2/components/appointment-header/template.jinja

```diff
@@ -5,7 +5,7 @@
 
 <p>{{ clinic_slot.slot_time_and_clinic_date }}{% if not appointment.current_status.is_screened %}<span class="nhsuk-u-margin-left-3"><a href="#">Reschedule appointment</a></span>{% endif %}</p>
 
-{% if not appointment.extra_needs and not appointment.current_status.is_screened %}
+{% if not appointment.is_special_appointment and not appointment.current_status.is_screened %}
```

Comment:
Could this be a tested method on appointment? `.can_be_made_special` or something like that?

Reply:
@gpeng I've pushed another commit with this in. Also extracted css classes from the inline styles.